src/plugins/lens_oss/public/plugin.ts

```diff
@@ -17,25 +17,20 @@
  * under the License.
  */
 
-import { EuiIcon, IconType } from '@elastic/eui';
-import React from 'react';
+import { CoreSetup } from 'kibana/public';
+import { VisualizationsSetup } from 'src/plugins/visualizations/public';
+import { getLensAliasConfig } from './vis_type_alias';
 
-interface VisTypeIconProps {
-  icon?: IconType;
-  image?: string;
+export interface LensPluginSetupDependencies {
+  visualizations: VisualizationsSetup;
 }
 
-/**
- * This renders the icon for a specific visualization type.
- * This currently checks the following:
- * - If image is set, use that as the `src` of an image
- * - Otherwise use the icon as an EuiIcon or the 'empty' icon if that's not set
- */
-export const VisTypeIcon = ({ icon, image }: VisTypeIconProps) => {
-  return (
-    <React.Fragment>
-      {image && <img src={image} alt="" className="visNewVisDialog__typeImage" />}
-      {!image && <EuiIcon type={icon || 'empty'} size="l" color="secondary" />}
-    </React.Fragment>
-  );
-};
+export class LensOSSPlugin {
+  constructor() {}
```

Comment:
I don't know why github shows the difference between this file and the `vis_type_icon` which is deleted. It is a new file on a new folder 🙂

Reply:
I suppose that's because of license part at the top. It treats this as `vis_type_icon.tsx` (which was removed) which changed location and content a bit 😃 
I faced this a lot.

Btw, nit: I think you could skip the ` constructor() {}` line here

Reply:
oh that makes sense! I will dear, thank you 😊